api/config.py

```diff
@@ -43,7 +43,9 @@ class Configuration(CoreConfiguration):
     OAUTH_TOKEN_EXPIRATION_DAYS = 'token_expiration_days'
     SECRET_KEY = "secret_key"
 
-    MILLENIUM_INTEGRATION = u"Millenium"
+    MILLENIUM_INTEGRATION = "Millenium"
```

Comment:
Are these two settings still required? The only places I see them used are here and -- for `MILLENIUM_INTEGRATION` in the integration_tests/__init__.py setup (which could be switched out for an external integration.